t/16-utils.t

```diff
@@ -61,6 +62,19 @@ subtest 'random number generator' => sub {
     isnt($r, $r2, "random_hex produces different results");
 };
 
+subtest 'download speed' => sub {
+    is download_speed([1638459407, 528237], [1638459408, 628237], 1024), '930.91 Byte/s';
+    is download_speed([1638459407, 528237], [1638459408, 628237], 1024 * 1024), '931 KiB/s';
```

Comment:
Unless `human_readable_size` lacks test coverage for these cases we likely don't need to test all these cases for the download speed as well.

Reply:
I don't agree - but I wonder if this can't be simplified with a hash

Reply:
In code a hash is usually the right choice, but for tests i really don't like it. Because it makes debugging a bit harder for various reasons. For example you can't as easily rerun one of the cases to replicate a problem, or in the case of an error the line number will point at the test loop, not the actual case that failed. You get the idea. :)